Dockerfile

```diff
@@ -1,7 +1,7 @@
-FROM node:16-slim as BASE
+FROM node:18-slim as BASE
 
 RUN apt-get update && apt-get install -y wget && \
-    wget -qO- https://raw.githubusercontent.com/anchore/syft/main/install.sh | sh -s -- -b /usr/local/bin && \
+    wget -qO- https://raw.githubusercontent.com/anchore/syft/main/install.sh | sh -s -- -b /usr/local/bin v0.93.0 && \
```

Comment:
When docker build is ran from GithubActions it doesn't have cached the version, was working on my machine because it was cached and I was always building with the version I had (initially it was 0.93.0), in the other hand the github runner always downloaded the latest which had breaking changes.,

Reply:
pull that up to a arg so it is easy to spot/change, e.g.
`ARG SyftVersion=v0.93.0`